src/status_im/contexts/wallet/common/activity_tab/events.cljs

```diff
@@ -4,12 +4,19 @@
             [utils.re-frame :as rf]
             [utils.transforms :as transforms]))
 
+(defonce ^:private request-id-atom (atom 0))
+
+(defn- get-unique-request-id
+  []
+  (swap! request-id-atom inc)
+  @request-id-atom)
```

Comment:
This request id is using a simple counter, is this local only?

Reply:
Yeah, we only need it to be locally unique, and no two somoultaneous request to have the same id. This technique is good enough to ensure it